src/renderer/dx/DxRenderer.cpp

```diff
@@ -295,6 +295,17 @@ DxEngine::~DxEngine()
     RETURN_IF_FAILED(_d2dRenderTarget->CreateSolidColorBrush(D2D1::ColorF(D2D1::ColorF::White),
                                                              &_d2dBrushForeground));
 
+    D2D1_STROKE_STYLE_PROPERTIES strokeStyleProperties{
+        D2D1_CAP_STYLE_SQUARE,

```

Comment:
I know I can go look it up, but maybe leave comments on the end of the line explaining what each of the params do? Sometimes we do that for param heavy things.